R/derive_extreme_event.R

```diff
@@ -559,9 +567,22 @@ event <- function(dataset_name = NULL,
 #'   `derive_extreme_event()`. If the argument is not specified, the input
 #'   dataset (`dataset`) of `derive_extreme_event()` is used.
 #'
+#'   *Permitted Values*: a character scalar
+#'
 #' @param condition An unquoted condition for selecting the observations, which
 #'   will contribute to the extreme event.
 #'
+#'   The condition is applied to the joined dataset for selecting the confirmed
+#'   observations. The condition can include summary functions. The joined
```

Comment:
should we list some of the possible summary function like you did in the NEws?

will all of them work? 
![image](https://github.com/pharmaverse/admiral/assets/10111024/8893ee4d-33af-4918-8f45-113d4d6763c1)

Reply:
Yes, good idea. I have added them.

Yes, all should work.